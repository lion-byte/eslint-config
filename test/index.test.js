const config = require('../index')

describe('ESLint configuration', () => {
  test('should exist', () => {
    expect(config).toMatchSnapshot()
  })
})
