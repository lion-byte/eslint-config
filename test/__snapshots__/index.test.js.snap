// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ESLint configuration should exist 1`] = `
Object {
  "extends": Array [
    "plugin:jest/recommended",
    "standard",
    "standard-jsx",
  ],
  "parserOptions": Object {
    "ecmaVersion": 11,
  },
  "plugins": Array [
    "jest",
    "jsx-a11y",
    "react-hooks",
  ],
  "rules": Object {
    "no-var": "error",
    "react/jsx-uses-react": "error",
    "react/jsx-uses-vars": "error",
  },
}
`;
